### PR TITLE
Add githubteacher San Diego file with recs

### LINKED ISSUE #24: Create a file for githubteacher San Diego, CA
- [x] Create a branch
- [x] Add commit
- [x] Open a PR
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by githubteacher**
closes #24

### CODE REVIEW

**Comment by ISalzman** [comment]
:8ball:

File: githubteacher-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# San Diego, CA
+
+### Places to Eat
+- City Tacos
+- Lucha Libre
+
+### Places to Go
```
Comment (ISalzman) [inline githubteacher-sandiego.md]: What about Sea World?

Comment (allthedoll) [reply githubteacher-sandiego.md]: I disagree, @ISalzman, they trap whales and stuff.

File: githubteacher-sandiego.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# San Diego, CA
+
+### Places to Eat
+- City Tacos
+- Lucha Libre
+
```
Comment (allthedoll) [inline githubteacher-sandiego.md]: City Tacos is mucho deliciouso, but I think Pokéz is also wonderful.

Comment (githubteacher) [reply githubteacher-sandiego.md]: I haven't been there, thanks for the invite @allthedoll

File: githubteacher-sandiego.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+### Places to Eat
+- City Tacos
+- Lucha Libre
+- Rustic Root
+- NEW RESTAURANT
+
+### Places to Go
+- Coronado
+- Balboa Park
+- Sea World
```
Comment (ISalzman) [inline githubteacher-sandiego.md:12]: :clap: Thanks for including my recommendation.

File: githubteacher-sandiego.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# San Diego, CA
+
+### Places to Eat
+- City Tacos
+- Lucha Libre
+- Rustic Root
+- NEW RESTAURANT
```
Comment (ISalzman) [inline githubteacher-sandiego.md:7]: :question: Missing something?

**Review by ISalzman — DISMISSED** [review]

**Review by justin-silveri — APPROVED** [review]

**Pull request merged**